3-enrich/scala-common-enrich/src/test/scala/com.snowplowanalytics.snowplow.enrich.common/enrichments/miscEnrichmentSpecs.scala

```diff
@@ -37,7 +37,7 @@ class EtlVersionSpec extends MutSpecification {
 
   "The ETL version" should {
     "be successfully returned" in {
-      MiscEnrichments.etlVersion("hadoop-0.14.0") must_== "hadoop-0.14.0-common-0.18.0"
+      MiscEnrichments.etlVersion("hadoop-1.4.0") must_== "hadoop-1.4.0-common-0.19.0"
```

Comment:
In R74, we simply updates the hadoop-x.x.x to the latest available hadoop version